packages/app/src/views/Home.vue

```diff
@@ -20,15 +20,18 @@
           @closed="selectedTag = null"
         >
           <div class="flex flex-row content-start overflow-auto">
-            <TagEntry
-              v-for="entry in group.tags"
-              :key="entry.nfcData"
-              class="m-4 w-2/6 flex-shrink-0 text-4xl"
-              :class="{ 'opacity-25': selectedTag !== entry && selectedTag !== null }"
-              :img="entry.imageUrl"
-              :name="entry.name"
-              @click="toggleSelectedTag(entry)"
-            />
+            <div v-for="tag in group.tags" :key="tag.nfcData" class="pb-3">
+              <TagEntry
+                class="mx-4 w-44 flex-shrink-0 text-2xl"
+                :class="{ 'opacity-25': selectedTag !== tag && selectedTag !== null }"
+                :img="tag.imageUrl"
+                @click="toggleSelectedTag(tag)"
+              />
+              <div class="pl-5 text-white text-lg font-semibold">
+                {{ tag.name }}
+                <div class="font-thin italic text-sm">{{ tag.trackTitle }}</div>
+              </div>
+            </div>
```

Comment:
At the moment you cant see that is scrollable

Reply:
depends on your screen ratio

Reply:
could you send a screenshot of the problem?
this is what I am seeing:

![image](https://user-images.githubusercontent.com/9517685/106140283-db641c00-616e-11eb-9138-2721ff600ada.png)
![image](https://user-images.githubusercontent.com/9517685/106140336-ec149200-616e-11eb-9e13-b5d4f2565d8b.png)

Reply:
It appears only on small screens e.g Pixel 2